boards/shields/mikroe_adc_click/boards/lpcxpresso55s16_ns.overlay

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2020 Henrik Brix Andersen <henrik@brixandersen.dk>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/delete-node/ &mcp3204;
+
+&mikrobus_spi {
+	status = "okay";
+
+	/* LPCXpresso55S16 uses SSEL1 for mikroBUS SPI */
+	mcp3204: mcp3204@1 {
```

Comment:
@galak I found no way of changing the unit address from a board-specific devicetree overlay, hence I deleted the original node and re-added it using another unit address.

Is there a way of changing the unit address of an existing node without having to redefine the entire node?

Reply:
There isn't its a long standing issue w/dtc.